assembly/cli/src/main/java/org/eclipse/sw360/antenna/frontend/cli/AntennaCLIOptions.java

```diff
@@ -122,18 +162,20 @@ public boolean equals(Object o) {
             return false;
         }
         AntennaCLIOptions that = (AntennaCLIOptions) o;
-        return Objects.equals(getConfigFilePath(), that.getConfigFilePath());
+        return Objects.equals(getConfigFilePath(), that.getConfigFilePath())
+                && Objects.equals(getPropertiesFilePath(), that.getPropertiesFilePath());
     }
 
     @Override
     public int hashCode() {
-        return Objects.hash(super.hashCode(), getConfigFilePath());
+        return Objects.hash(super.hashCode(), getConfigFilePath(), getPropertiesFilePath());
     }
 
     @Override
     public String toString() {
         return "AntennaCLIOptions{" +
                 "configFilePath='" + configFilePath + '\'' +
+                "propertiesFilePath='" + propertiesFilePath + '\'' +
```

Comment:
very good, the propertiesFile should also be reflected in the helpMessage: https://github.com/eclipse/antenna/blob/80df3ad50475177ed57537286dc574cca3b95cd6/assembly/cli/src/main/java/org/eclipse/sw360/antenna/frontend/cli/AntennaCLIOptions.java#L144